src/python/pants/engine/rules_test.py

```diff
@@ -771,39 +766,6 @@ def a_from_suba(suba: SubA) -> A:
             fullgraph,
         )
 
-    def test_full_graph_for_planner_example(self):
```

Comment:
I don't think there's much value from this test. It's brittle because it depends on the state of the rest of the codebase, i.e. how many rules are registered in `create_fs_rules` and `create_graph_rules`. We also test this functionality elsewhere in the file.